app/helper.js

```diff
@@ -309,7 +321,7 @@ class YoutubeGrabberHelper {
     const communityTab = YoutubeGrabberHelper.findTab(contentDataJSON.contents.twoColumnBrowseResultsRenderer.tabs)
 
     if (communityTab) {
-      contentDataJSON = contentDataJSON.contents.twoColumnBrowseResultsRenderer.tabs[3].tabRenderer.content.sectionListRenderer.contents[0].itemSectionRenderer
+      contentDataJSON = contentDataJSON.contents.twoColumnBrowseResultsRenderer.tabs.find((tab) => tab.tabRenderer?.title === "Community").tabRenderer.content.sectionListRenderer.contents[0].itemSectionRenderer
```

Comment:
You can use the `communityTab` variable instead of finding the tab again.
```suggestion
      contentDataJSON = communityTab.tabRenderer.content.sectionListRenderer.contents[0].itemSectionRenderer
```

Reply:
🤦 thank you for pointing that out